### ISSUE TITLE
Calling onInputsChange when disabled

### ISSUE THREAD

**Issue by davedoesdev** [torch2424/responsive-gamepad#24]
What do you think should happen if `onInputsChange()` is called before `enable()` is called?

At the moment it starts the interval running.

So if I don't want the interval on my page until I enable `ResponsiveGamepad`, I have to remember whether it's the first time I've enabled it and only call `onInputsChange()` then.

What would be nicer is if I could call `onInputsChange()` before calling `enable()` and be sure that no interval has yet been started.

I think it's a one-liner:

```diff
diff --git a/lib/index.js b/lib/index.js
index 32ea84a..ee70728 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -190,7 +190,7 @@ class ResponsiveGamepadService {
     };
 
     // Check if we need to start the interval
-    if (!this.cancelInputChangeListener) {
+    if (this._enabled && !this.cancelInputChangeListener) {
       this._startInputChangeInterval();
     }
     
```

**Comment by torch2424** [comment]
@davedoesdev Ah that is a good question! 👍 

So what you proposed makes sense. As not enabling the gamepad shouldn't do anything. But if you do the `onInputsChange()` before enabling in your fix. Enabling would not start the interval. Would you want to keep a record of some sort of what intervals need to be set whenever the gamepad is enabled?

Thanks! 😄

**Comment by davedoesdev** [comment]
Won't it get started by `enable()` [here](https://github.com/torch2424/responsive-gamepad/blob/master/lib/index.js#L50)?

**Comment by torch2424** [comment]
@davedoesdev Oh duh, you are completely correct 😂 🙃 

I wrote this code a little while ago, thus I forgot. Yeah this all sounds great then! Would definitely pull this in 👍 Feel free to open a PR, or I can do it once I get the time 😄

**Comment by davedoesdev** [comment]
https://github.com/torch2424/responsive-gamepad/pull/25

**Comment by torch2424** [comment]
This was closed by #25 😄
